### PR TITLE
docs: Add note to releases about stepping through versions

### PR DESCRIPTION
**Pull request by jszwedko**
To encourage users to step through minor releases.

Signed-off-by: Jesse Szwedko <jesse@szwedko.me>

### CODE REVIEW

**Review by spencergilbert — APPROVED** [review]
Ah, had we not specified this before?

**Comment by jszwedko** [comment]
> Ah, had we not specified this before?

Not explicitly, that I could find.

**Comment by spencergilbert** [comment]
> Not explicitly, that I could find.

I also don't remember seeing it anywhere, just talking about it - definitely good to have it included now

File: website/layouts/releases/single.html [hunk 1]
```diff
@@ -32,6 +32,29 @@
       </div>
       {{ end }}
 
+      <div class="mt-8">
```
Comment (jszwedko) [inline website/layouts/releases/single.html:35]: Borrowed this from the shortcode for warnings (`./website/layouts/shortcodes/warning.html`).

**Pull request merged**